Travis failure:
```
> Task :spring-integration-core:checkstyleMain
[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-core/src/main/java/org/springframework/integration/metadata/PropertiesPersistingMetadataStore.java:101:49: '&&' is not followed by whitespace. [WhitespaceAround]
```

--- a/spring-integration-core/src/main/java/org/springframework/integration/metadata/PropertiesPersistingMetadataStore.java
+++ b/spring-integration-core/src/main/java/org/springframework/integration/metadata/PropertiesPersistingMetadataStore.java
@@ -98,7 +98,7 @@ public class PropertiesPersistingMetadataStore implements ConcurrentMetadataStor
 		}
 		this.file = new File(baseDir, this.fileName);
 		try {
-			if (!this.file.exists() &&!this.file.createNewFile() && this.logger.isWarnEnabled()) {
+			if (!this.file.exists() && !this.file.createNewFile() && this.logger.isWarnEnabled()) {
 				this.logger.warn("Failed to create file " + this.file);
 			}
 		}

